src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.ts

```diff
@@ -31,12 +31,20 @@ export class EditSpaceDescriptionWidgetComponent implements OnInit {
     private userService: UserService,
     private broadcaster: Broadcaster,
     private spaceService: SpaceService,
-    private spaceNamespaceService: SpaceNamespaceService
+    private spaceNamespaceService: SpaceNamespaceService,
+    private collaboratorService: CollaboratorService
   ) {
-    spaces.current.subscribe(val => {
-      this.space = val;
-      console.log('newspace', val);
-    });
+    spaces.current
+      .subscribe(space => {
+        this.space = space;
```

Comment:
Are there any consequences for setting space even if the incoming space is null or undefined?

Reply:
There shouldn't be, all the instances of checking/using `space` [0][1][3][4][5][6][7][8] take into the account the possibility of the space not having a value.

Except for [2], but I've added a check in the latest commit that moves the spaces subscription to the ngOnInit

[0] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.ts#L50
[1] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.ts#L51
[2] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.ts#L54
[3] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.ts#L64
[4] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.html#L4
[5] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.html#L15
[6] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.html#L26
[7] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.html#L41
[8] https://github.com/fabric8-ui/fabric8-ui/blob/master/src/app/dashboard-widgets/edit-space-description-widget/edit-space-description-widget.component.html#L45